docs/concepts/managed-resources.md

```diff
@@ -9,42 +9,202 @@ indent: true
 
 ## Overview
 
-Managed resources are the Crossplane representation of the cloud
-[provider][provider] resources and they are considered primitive low level
-custom resources that can be used directly to provision external cloud resources
-for an application or as part of an infrastructure composition.
-
-For example, `RDSInstance` in AWS Provider corresponds to an actual RDS Instance
-in AWS. There is a one-to-one relationship and the changes on managed resources
-are reflected directly on the corresponding resource in the provider.
-
-You can browse [API Reference][api-reference] to discover all available managed
-resources.
+A Managed Resource is Crossplane's representation of a resource in an external
+system - most commonly a cloud provider. Managed Resources are opinionated, XRM
+compliant Kubernetes Custom Resources that are installed by a Crossplane
+[provider].
+
+For example, `RDSInstance` in the AWS Provider corresponds to an actual RDS
+Instance in AWS. There is a one-to-one relationship and the changes on managed
+resources are reflected directly on the corresponding resource in the provider.
+Similarly, the `Database` types in the SQL provider represent a PostgreSQL or
+MySQL database. You can browse [API Reference][api-reference] to discover all
+available managed resources.
+
+Managed Resources are the building blocks of Crossplane. They're designed to be
+_composed_ into higher level, opinionated Custom Resources that Crossplane calls
+Composite Resources or XRs - not used directly. See the
+[Composition][composition] documentation for more information.
 
 ## Syntax
 
 Crossplane API conventions extend the Kubernetes API conventions for the schema
-of Crossplane managed resources. Following is an example of `RDSInstance`:
+of Crossplane managed resources. Following is an example of a managed resource:
+
+<ul class="nav nav-tabs">
+<li class="active"><a href="#aws-tab-1" data-toggle="tab">AWS</a></li>
+<li><a href="#gcp-tab-1" data-toggle="tab">GCP</a></li>
+<li><a href="#azure-tab-1" data-toggle="tab">Azure</a></li>
+</ul>
+<br>
+<div class="tab-content">
+<div class="tab-pane fade in active" id="aws-tab-1" markdown="1">
+
+The AWS provider supports provisioning an [RDS] instance via the `RDSInstance`
```

Comment:
Most of `managed-resources.md` is just two documents I merged together. I would prefer to make any non-trivial updates to the content of `managed-resources.md` in a future PR.

That said, my personal preference is to keep the details on how to install providers under the section where that already exists - I think it would quickly become repetitive and difficult to maintain if we tried to supply all the prior context needed to get to a state inline.